features/step_definitions/borrowers_personal_info.rb

```diff
@@ -0,0 +1,9 @@
+Given(/^I add the following deed:$/) do |deed|
+  deed_end_point = Env.deed_api + '/deed/'
+  @response = HTTP.post(deed_end_point, json: JSON.parse(deed))
+  puts @response
```

Comment:
You will want to remove this puts @response from the code as it's not really necessary for the test and it's good to keep the code clean and running as efficiently as possible.